Managers/UInputMan.cpp

```diff
@@ -1,2858 +1,1393 @@
+						}
+					}
+					if (element[INPUT_L_UP].JoyDirMapped()) {
+						if (joy[joystick].stick[element[INPUT_L_UP].GetStick()].flags & JOYFLAG_UNSIGNED) {
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].pos = 128;
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].d2 = 0;
+						} else {
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].pos = 0;
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_L_UP].GetStick()].axis[element[INPUT_L_UP].GetAxis()].d2 = 0;
+						}
+					}
+				}
+
+				aimValues.Reset();
+
+				// Right stick
+				if (element[INPUT_R_LEFT].JoyDirMapped()) { aimValues.m_X = AnalogAxisValue(joystick, element[INPUT_R_LEFT].GetStick(), element[INPUT_R_LEFT].GetAxis()); }
+				if (element[INPUT_R_UP].JoyDirMapped()) { aimValues.m_Y = AnalogAxisValue(joystick, element[INPUT_R_UP].GetStick(), element[INPUT_R_UP].GetAxis()); }
+
+				if (aimValues.GetMagnitude() < deadZone * 2) {
+					if (element[INPUT_R_LEFT].JoyDirMapped()) {
+						if (joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].flags & JOYFLAG_UNSIGNED) {
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].pos = 128;
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].d2 = 0;
+						} else {
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].pos = 0;
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_R_LEFT].GetStick()].axis[element[INPUT_R_LEFT].GetAxis()].d2 = 0;
+						}
+					}
+					if (element[INPUT_R_UP].JoyDirMapped()) {
+						if (joy[joystick].stick[element[INPUT_R_UP].GetStick()].flags & JOYFLAG_UNSIGNED) {
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].pos = 128;
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].d2 = 0;
+						} else {
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].pos = 0;
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].d1 = 0;
+							joy[joystick].stick[element[INPUT_R_UP].GetStick()].axis[element[INPUT_R_UP].GetAxis()].d2 = 0;
+						}
+					}
+				}
+			}
+
+			// Stick, axis, directions
```

Comment:
What does this comment mean to you? Cause it means shit all to me!

Reply:
¯\\\_( ͡° ͜ʖ ͡°)_/¯

Reply:
KILL IT!